README.md

```diff
@@ -252,7 +252,8 @@ const Button = styled("button", {
   },
 });
 
-type ButtonTypes = CSSComponentPropType<typeof Button, "primary">;
+type ButtonVariants = CSSComponentPropType<typeof Button>;
```

Comment:
The CSSComponentPropType import above was removed, would this line still be valid?

Reply:
Good catch!